.travis.yml

```diff
@@ -22,12 +22,18 @@ before_script:
   - echo "include ':app', ':commons'" > 'settings.gradle'
   - sed -i.bak '150,163d' $FLAKY_PATH
 
-env:
-  - TEST_SUITE=integration_tests
-  - TEST_SUITE=unit_tests
-
-script: ./testing $TEST_SUITE
+jobs:
+  include:
+    - stage: Compilev (Debug, UnitTest, AndroidTest & Release) 🌚
```

Comment:
Not sure if use of emoticons (🌚) follows the coding conventions.

Reply:
It matches travis conventions: "Stages are identified by their names, which are composed of names and emojis." (Source: [docs.travis-ci.com](https://docs.travis-ci.com/user/build-stages/))

Reply:
@0matleb2 I had to do it 😉